docs/intents/ref/CreateInteraction.md

```diff
@@ -0,0 +1,69 @@
+---
+id: CreateInteraction
+sidebar_label: CreateInteraction
+title: CreateInteraction
+hide_title: true
+---
+# `CreateInteraction`
+
+Create an interaction with a list of contacts.
+
+## Intent Name
+
+`CreateInteraction`
+
+## Display Name
+
+`Create Interaction`
+
+## Possible Contexts
+
+* [ContactList](../../context/ref/ContactList)
+
+## Example
+
+```js
+const interaction = {
+    type: 'fdc3.interaction',
+    contacts: {
+        type: 'fdc3.contactList',
+        contacts: [
+            {
+                type: 'fdc3.contact',
+                name: 'Jane Doe',
+                id: {
+                    email: 'jane.doe@mail.com'
+                }
+             },
+            {
+                type: 'fdc3.contact',
+                name: 'John Doe',
+                id: {
+                    email: 'john.doe@mail.com'
+                }
+            },
+        ]
+    },
+    interactionType: 'Instant Message',
+    timeRange: {
+        type: 'fdc3.timeRange',
+        startTime: '2022-02-10T15:12:00Z'
+    },
+    description: 'Laboris libero dapibus fames elit adipisicing eu, fermentum, dignissimos laboriosam, erat, risus qui deserunt. Praesentium! Reiciendis. Hic harum nostrud, harum potenti amet? Mauris. Pretium aliquid animi, eget eiusmod integer proident. Architecto ipsum blandit ducimus, possimus illum sunt illum necessitatibus ab litora sed, nonummy integer minus corrupti ducimus iste senectus accumsan, fugiat nostrud? Pede vero dictumst excepturi, iure earum consequuntur voluptatum',
+    initiator:  {
+        type: 'fdc3.contact',
+        name: 'Jane Doe',
+        id: {
+            email: 'jane.doe@mail.com'
+        }
+    },
+    origin: 'Outlook'
+}
+
+fdc3.raiseIntent('CreateInteraction', interaction)
```

Comment:
Hi @kriswest I've just submitted another change that includes an example TransactionResult for CreateInteraction. 

The Interaction context didn't have an id property and so I've introduced one. This should allow a system that creates an interaction to return an id as part of the TransactionResult. 

This change is pending internal review, but please feel free to take a look and let me know what you think.

Reply:
Hi @kriswest - after going through this with @pauldyson, I've updated Interaction.md to support application specific identifiers (e.g.- id.SINGLETRACK, id.SALESFORCE). 

The example TransactionResult in CreateInteraction.md uses a Singletrack ID. 

I've also added 'id.URI' so that an application can pass a link to an interaction record. Although Singletrack automatically opens its interaction logger when an interaction is created, other applications may not. Returning a URI would give an originating application the flexibility to open the created record. 

Happy to discuss further on this Thursday's call.